bin/generate-travis-yml.sh

```diff
@@ -19,35 +19,52 @@ cd $TIMESTAMP
 # TODO: make this configurable later
 cat <<EOT >> .travis.yml
 language: node_js
-sudo: false
 node_js:
```

Comment:
Was there a reason you needed to take this out? I was under the impression that this forced us to use containers, which would speed up builds.